### PR TITLE
Feat/add faq section

### LINKED ISSUE #10: Create Frequently Asked Questions Section
## 📄 Issue Description
Implement the "Frequently Asked Questions" (FAQ) section of the landing page based on the Figma design. This section provides answers to common user queries in an organized and user-friendly layout.

## 🔍 Steps

1. Analyze the Figma design for the FAQ section.
2. Review the structure of existing components for alignment.
3. Create a reusable accordion component for questions and answers.
4. Style the component to match the theme of the page.
5. Test for responsiveness and accessibility across devices.

## ✅ Acceptance Criteria

- The FAQ section matches the Figma design.
- Questions and answers are presented in a collapsible accordion format.
- Section is fully responsive and accessible.
- Components must be separated in their individual folder per section. In this case, use the faq folder in components.

## 🌎 References
[Figma Design](https://github.com/PayStell/paystell-landing/blob/main/landingPage.md)

### PR DESCRIPTION
**Pull request by zleypner**
# Pull Request Overview

The FAQ section features:
- Clean, modern design matching Figma specifications
- Smooth accordion animations for question/answer pairs
- Responsive layout that adapts to all screen sizes
- Proper spacing and typography following design system
- Interactive elements with hover and focus states
- Semantic HTML structure for accessibility

image

## 🧪 Testing
Testing completed:
- ✅ Verified component rendering in all major browsers
- ✅ Tested responsive behavior across different screen sizes
- ✅ Validated accordion functionality
- ✅ Confirmed proper TypeScript compilation
- ✅ Checked accessibility using keyboard navigation
- ✅ Verified component integration with Next.js

## 💬 Comments
- The component uses client-side rendering due to its interactive nature
- Implementation follows the atomic design pattern
- All text content is currently hardcoded but can be easily modified for localization if needed
- Styling uses Tailwind utility classes for consistency with the rest of the project

This PR closes #10

### CODE REVIEW

**Comment by zleypner** [comment]
Hello @MPSxDev !  Would you like to update the information once the faq is clicked for each of the questions? If you do, please provide me with the content. I will be more than glad to proceed with the updates or additional request you may have.

**Comment by MPSxDev** [comment]
> Hello @MPSxDev ! Would you like to update the information once the faq is clicked for each of the questions? If you do, please provide me with the content. I will be more than glad to proceed with the updates or additional request you may have.

Here is the file with the questions and answers. Thank you. 
[FAQ PayStell.pdf](https://github.com/user-attachments/files/18136963/FAQ.PayStell.pdf)

**Comment by zleypner** [comment]
Hello @MPSxDev ! Thank you for sharing the FAQ content. If you have any additional request I will be more than glad to proceed with the updates.

image

File: src/components/sections/FAQ.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  };
+
+  return (
+    <section id="faq" className="py-24 bg-background">
+      <div className="container">
+        <div className="max-w-3xl mx-auto">
+          <div className="flex items-center gap-2 text-sm text-primary mb-4">
+            <span className="font-medium">SUPPORT</span>
+            <span className="text-lg">★</span>
+          </div>
```
Comment (MPSxDev) [inline src/components/sections/FAQ.tsx:52]: Use Badge
```suggestion
          <Badge className="w-fit bg-primary/10 px-3 rounded-xl hover:bg-primary/10">
            <h2 className="text-lg font-bold text-primary mr-3">SUPPORT</h2>
            <Image src="/star.svg" width={28} height={28} alt="star" />
          </Badge>
```

File: src/components/sections/FAQ.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
+"use client"
+
+import { ChevronDown } from "lucide-react";
+import { useState } from "react";
+
```
[the remark this answers is not in the document]
Comment (zleypner) [reply src/components/sections/FAQ.tsx:5]: @MPSxDev , Thank you , I will update asap

**Review by MPSxDev — APPROVED** [review]
Excellent work, thank you so much!

**Pull request merged**